arm/redis/firewallrule.go

```diff
@@ -1,254 +0,0 @@
-package redis
```

Comment:
Deleted.

Reply:
Checked the redis swagger, firewall stuff is included in a [previous](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/redis/resource-manager/Microsoft.Cache/2016-04-01/redis.json) version, but not in [latest](https://github.com/Azure/azure-rest-api-specs/tree/current/specification/redis/resource-manager/Microsoft.Cache/2017-02-01) version

Reply:
I think we should move back to the previous version as this would be a pretty big breaking change and I see no mention that the firewall stuff has been deprecated (we should follow up with the redis cache folks to understand what's going on here).

Reply:
New PR generated with old version https://github.com/Azure/azure-sdk-for-go/pull/788